cirq-google/cirq_google/engine/engine_client.py

```diff
@@ -122,9 +130,10 @@ def __init__(
 
         self._service_args = service_args
 
-    @cached_property
```

Comment:
Oops, need to add back `@property` since other places in the code call this like a property instead of a method.

Reply:
Makes sense. Fixed and tested; thanks!